draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -2099,3 +2101,5 @@ Specification document:
 This document is a minor update of RFC 6265, adding small features, and
 aligning the specification with the reality of today's deployments. Here,
 we're standing upon the shoulders of giants.
+
+Thanks to Adam Barth for starting the work on this update.
```

Comment:
This doesn't seem like enough. :) Would you mind reformulating this section to be something similar to RFC7159's https://tools.ietf.org/html/rfc7159#section-14?